airflow/example_dags/example_params_trigger_ui.py

```diff
@@ -27,7 +27,7 @@
 
 from airflow.decorators import task
 from airflow.models.dag import DAG
-from airflow.models.param import Param, ParamsDict
+from airflow.sdk.definitions.param import Param, ParamsDict
```

Comment:
Same thing here, lets make this work in dags

```suggestion
from airflow.sdk import Param, ParamsDict
```

Reply:
Ah you already have it working, lets use it then

Reply:
Yeah, I will let it be

Reply:
Example dags are what a lot of users use over the docs, so we should use what we want users to do -so let's change this and any other examples

Reply:
Yeah that makes good sense, let me do that.